### ISSUE TITLE
CSS rules for multi-input clash with Bootstrap's form control rules

### ISSUE THREAD

**Issue by evrimfeyyaz** [ericgio/react-bootstrap-typeahead#542]
When you import Typeahead's CSS file before Bootstrap's, the `height` property of Bootstrap's `.form-control` rule takes precedence over the one in Typeahead's `.rbt-input-multi`.

```HTML
<div class="rbt-input-multi form-control rbt-input" tabindex="-1">
    <!-- ... -->
</div>
```

```CSS
.rbt-input-multi {
    cursor: text;
    overflow: hidden;
    position: relative;
    height: auto; // <-- This doesn't work.
}

.form-control {
    display: block;
    width: 100%;
    height: calc(1.5em + 0.75rem + 2px);
    padding: 0.375rem 0.75rem;
    font-size: 1rem;
    font-weight: 400;
    line-height: 1.5;
    color: #495057;
    background-color: #fff;
    background-clip: padding-box;
    border: 1px solid #ced4da;
    border-radius: 0.25rem;
    transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
}
```

### Version
5.0.0-alpha.1

### Steps to reproduce
Include Typeahead's CSS file before Bootstrap's.

### Expected Behavior
Typeahead input's height should grow when more elements added.

<img width="286" src="https://user-images.githubusercontent.com/11249734/80869831-a6946880-8cb3-11ea-800f-4ece63cbd316.png">

### Actual Behavior
The height stays the same.

<img width="278" src="https://user-images.githubusercontent.com/11249734/80869845-b6ac4800-8cb3-11ea-8769-f58eea97ce80.png">

**Comment by ericgio** [comment]
Hi @evrimfeyyaz and thanks  for raising this issue. What you describe makes sense, though I'm not sure it's a bug. The assumption is that the Typeahead CSS are imported after Bootstrap. Not sure what the right solution is. I could clarify the docs to specify the above or increase the specificity of the CSS.

**Comment by evrimfeyyaz** [comment]
@ericgio I would personally suggest making the rule more specific. This is the rule I'm using in my own application:

```CSS
.rbt-input-multi.form-control {
  cursor: text;
  overflow: hidden;
  position: relative;
  height: auto;
}
```

**Comment by ericgio** [comment]
Sure, first solution seems reasonable. Can you submit a pull request?

**Comment by ericgio** [comment]
Released as part of v5.0.0-rc.1
